src/content/structured/accessibility/needs/general.mdx

```diff
@@ -0,0 +1,84 @@
+---
+path: "/accessibility/needs/general"
+
+navPriority: 1
+
+date: "2024-10-17"
+
+title: "Designing for accessibility needs"
+
+subTitle: "Every app and service you create must have a minimal baseline of accessible designs."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/accessibility/needs/general.mdx"
+---
+
+import generalAccessibilityFig1 from "./images/fig-1-linear-layout.png";
+import generalAccessibilityFig2 from "./images/fig-2-non-linear-layout.png";
+
+## Introduction
+
+<p>
+  The{" "}
+  <ic-link
+    target="_blank"
+    href="https://accessibility.blog.gov.uk/2016/09/02/dos-and-donts-on-designing-for-accessibility/"
+    rel="noreferer noopener nofollow"
+  >
+    Home Office
+  </ic-link>{" "}
+  has designed effective accessibility posters from which the accessibility
```

Comment:
Suggestion to change this sentence to - "The Home Office has created effective posters on designing for accessibility from which the accessibility needs section has taken inspiration" - and change the link text to be 'posters on designing for accessibility'. This link text better describes where the link is taking you.